src/doc/src/reference/environment-variables.md

```diff
@@ -334,11 +334,15 @@ let out_dir = env::var("OUT_DIR").unwrap();
 * `RUSTC`, `RUSTDOC` — the compiler and documentation generator that Cargo has
                        resolved to use, passed to the build script so it might
                        use it as well.
+* `CARGO_RUSTC_WRAPPER` — the `rustc` wrapper, if any, that Cargo is using.
```

Comment:
I think this'll need an update, and I think it also needs to list the new `RUSTC_VERSION*` vars?